src/containers/shared/components/Transaction/README.md

```diff
@@ -5,10 +5,8 @@ Each transaction has its properties defined in `./{TransactionName}/index.ts`.
 - **Description**: A React component that defines the "Description" portion of a transaction's "Detailed" tab.
 - **Simple**: A React component that defines the left column of a transaction's "Simple" tab.
 - **TableDetail**: A React component that defines the body of transaction on the account, ledger, token, or nft page.
-- **mapper**: A function which takes a transaction object and returns additional properties to map onto a transaction.
-
-  This is run when a transaction is received from the server and is useful for transforming complex properties derived
-  from nodes.
+- **parser**: A function which takes a transaction object and meta nodes. It returns additional properties to map onto a transaction.
```

Comment:
```suggestion
- **parser**: A function which takes a transaction object and (optionally) meta nodes. It returns additional properties to map onto a transaction.
```

Reply:
It always passes it in you just don't need to use it.